controllers/nnf_workflow_controller_test.go

```diff
@@ -63,7 +63,10 @@ var _ = Describe("NNF Workflow Unit Tests", func() {
 
 	AfterEach(func() {
 		Eventually(func() error {
-			Expect(k8sClient.Get(context.TODO(), key, workflow)).To(Succeed())
```

Comment:
I think the correct way to do this is to add an input parameter `g Ginkgo` and change the Expect to `g.Expect`

Sample from nnf-dm
```go
Eventually(func(g Gomega) dmv1alpha1.RsyncNodeDataMovementStatus {
	g.Expect(k8sClient.Get(context.TODO(), client.ObjectKeyFromObject(rsync), rsync)).To(Succeed())
	return rsync.Status
})
```

Reply:
Right. I'll change it

Reply:
Yes, please.
It's described in this section of the Gomega  doc: https://onsi.github.io/gomega/#eventually